tests/e2e/specs/shopper/filter-products-by-rating.test.ts

```diff
@@ -213,7 +230,7 @@ describe( `${ block.name } Block`, () => {
 
 			await insertBlock( 'Products (Beta)' );
 			await insertBlock( block.name );
-			await page.waitForNetworkIdle();
+			// await page.waitForNetworkIdle();
```

Comment:
We can probably remove this line.

Reply:
Removed with [6e39326](https://github.com/woocommerce/woocommerce-blocks/pull/9575/commits/6e3932689213d1db437f836b12922bbd951b1389)